### DISCUSSION TITLE
Welcome to RamHub Discussions!

### DISCUSSION THREAD

**Discussion by theshmunguss1** [theshmunguss1/RamHub#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WiFiGiver** [comment]
Finally, the best way to get to the hub!
